sema4ai/vscode-client/src/views.ts

```diff
@@ -86,6 +87,39 @@ export async function openPackageTreeSelection(robot?: RobotEntry) {
     }
 }
 
+export async function openRunbookTreeSelection(robot?: RobotEntry) {
+    const selectedRobot = robot ?? getSelectedRobot();
+
+    if (!selectedRobot) {
+        return;
+    }
+
+    const agentYamlPath = path.join(selectedRobot.robot.directory, "agent-spec.yaml");
+    const agentYamlUri = vscode.Uri.file(agentYamlPath);
+
+    if (!(await uriExists(agentYamlUri))) {
+        return;
+    }
+
+    const yamlContent = await vscode.workspace.fs.readFile(agentYamlUri);
```

Comment:
You shouldn't load yaml in the client side, rather create a command in the server side where you ask for the runbook path given an agent -- it's a bit more code, but should remove the added yaml dependency.

Reply:
Got it, will switch to an internal command for this then. Thanks